helm-charts/splunk-otel-collector/values.yaml

```diff
@@ -258,7 +258,12 @@ agent:
       # This value is being used as a source for default memory_limiter processor configurations
       memory: 500Mi
 
-  securityContext: {}
+  # To collect container logs and journald logs, it will run the agent as a root user.
+  # To run it as non root user, uncomment below `securityContext` options.
+
+  # securityContext:
```

Comment:
I curios if schema validation will fail if we remove `securityContext: {}` from here. I believe it should be failing because the type is required to be `object`, but now it's changed to `nil`

Reply:
schema validation doesn't fail with `securityContext: {}`. I think because `{}` is an object. `securityContext:` this fails tho.